pages/inspector/inspector-fix-issues.mdx

```diff
@@ -126,6 +126,47 @@ Once the necessary updates are made in the Tracking Plan, update the issue statu
   />
 </center>
 
+## Resolved issue status
+
+When setting an issue status to _Resolved_ ([see available issue statues](/inspector/inspector-issues-view#issue-status)), you have to specify for what version or what time point the issue should be resolved for.
+
+<center>
+  <img
+    src="/docs/images/inspector/inspector-resolved-issue-status.png"
+    width="600px"
+    alt="Image shows resolved issue options."
+  />
+</center>
+
+The available options are:
+- **Latest release**: The issue will be resolved for the latest release version that Inspector has seen when the issue is marked as resolved.
+- **Next release**: The issue will be resolved for the next release version that is incremented higher than the latest release version that Inspector sees. For example if the latest version is `2.2.1.` and the next version seen that is incremented higher is `2.2.2`, then the issue will be resolved for version `2.2.2` and every version after that.
```

Comment:
Maybe we note here that any "next" version higher than 2.2.1 will be validated, it does not need to be 2.2.2, as long as its higher than 2.2.1 it will be validated

Reply:
ahh yes I was looking at it wrong. Let me try to rephrase and see if I get it right.